Source/JavaScriptCore/wasm/WasmOperations.cpp

```diff
@@ -109,49 +109,70 @@ static void triggerOMGReplacementCompile(TierUpCount& tierUp, OMGCallee* replace
     }
 }
 
-void loadValuesIntoBuffer(Probe::Context& context, const StackMap& values, uint64_t* buffer)
+void loadValuesIntoBuffer(Probe::Context& context, const StackMap& values, uint64_t* buffer, unsigned valueSize)
 {
+    ASSERT(valueSize == 1 || valueSize == 2);
+    ASSERT(Options::useWebAssemblySIMD() || valueSize == 1);
+    bool usesSIMD = (valueSize == 2);
```

Comment:
This could be swapped I guess? Also, it would be nice to have some kind of helper for valueSize, but I  am not sure what to call it.